tooling/electron-mksnapshot/README.md

```diff
@@ -0,0 +1,27 @@
+## @packages/electron-mksnapshot
+
+A rewrite of [electron/mksnapshot](https://github.com/electron/mksnapshot) to support multiple
+versions.
+
+The main difference is that the _mksnapshot_ binary is not downloaded when installing this
+module. 
+
+Instead, whenever `electron-mksnapshot` is run an electron version to use can be
+provided or is resolved from the electron installed relative to the root of your project.
```

Comment:
> ... or is resolved from the electron installed relative to the root of your project.

Is this true? It looks like a provided `version` is required in the APIs.

Reply:
Good catch: [`359c383` (#23284)](https://github.com/cypress-io/cypress/pull/23284/commits/359c383c23e6637a1ffd7edca02a6c2f26596884)